### ISSUE TITLE
CMAKE error

### ISSUE THREAD

**Issue by DaScientist** [YDLIDAR/YDLidar-SDK#11]
The updated CMakeLists.txt is a file containing binary data, and not Cmake commands. Please resolve this as soon as possible

**Comment by k3street** [comment]
Agreed, this worked in a previous build. I am going to try that approach since  this worked a year ago.
